packages/lesswrong/server/migrations/2022-08-29-syncEmailsFields.ts

```diff
@@ -0,0 +1,58 @@
+import { registerMigration } from './migrationUtils';
+import Users from '../../lib/collections/users/collection';
+import fs from 'fs';
+import Papa from 'papaparse';
+
+registerMigration({
+  name: "syncEmailsFields",
+  dateWritten: "2022-08-29",
+  idempotent: true,
+  action: async () => {
+    const totalUsersMissingEmail =  await Users.find({email: null}).fetch()
+    const totalRecentUsersMissingEmail =  await Users.find({email: null, lastNotificationsCheck: {$exists:true}}).fetch()
```

Comment:
can't we just derive this from the results of `totalUsersMissingEmail` by checking which of the returned users are missing the `lastNotificationsCheck` field?